rotkehlchen/db/schema.py

```diff
@@ -848,6 +848,22 @@
 );
 """
 
+# Stores metrics for history events including balance, pnl, and cost_basis data.
+# Each row represents a metric for a specific bucket after the event is applied.
+# Bucket = (location, location_label, bucket_protocol, asset) where:
+# - bucket_protocol (NULL for wallet, derived from event's counterparty + type/subtype)
+# - metric_key is the type of metric ('balance', 'pnl', 'cost_basis', etc.)
+DB_CREATE_EVENT_METRICS = """
+CREATE TABLE IF NOT EXISTS event_metrics (
+    id INTEGER NOT NULL PRIMARY KEY,
+    event_identifier INTEGER NOT NULL REFERENCES history_events(identifier) ON DELETE CASCADE,
+    bucket_protocol TEXT,
```

Comment:
why bucket? I saw `Bucket = (location, location_label, bucket_protocol, asset)` But still not sure what it means?